### PR TITLE
alert MAT disclaimer

### PR DESCRIPTION
**Pull request by ObedVega**
**Alert**
image

### CODE REVIEW

File: libs/react-components/src/lib/components/Alert/Alert.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+type Story = StoryObj<typeof meta>;
+
+export const Basic: Story = {};
+
+export const ActionElements: Story = {
+  render: (args) => {
+    return (
+      <Alert {...args}>
+        <div slot="action-items">
+          <IconButton icon="close" />
```
Comment (owilliams320) [inline libs/react-components/src/lib/components/Alert/Alert.stories.tsx]: can we do without the div? it works but its simpler to just add the slot name to the single item here.

```suggestion
          <IconButton slot="action-items" icon="close" />
```

File: libs/react-components/src/lib/components/Alert/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.alertWrapper {
```
Comment (owilliams320) [inline libs/react-components/src/lib/components/Alert/styles.module.scss]: was this style needed?

**Review by owilliams320 — CHANGES_REQUESTED** [review]
Just a few small comments, but overall 👍🏽

File: libs/react-components/src/lib/components/Alert/styles.module.scss (hunk 2 above)
Comment (ObedVega) [reply libs/react-components/src/lib/components/Alert/styles.module.scss]: not need anymore

**Review by bsahitya — DISMISSED** [review]
LGTM! 🚀

**Review by owilliams320 — APPROVED** [review]

**Pull request merged**